src/ray/common/ray_config_def.h

```diff
@@ -375,3 +375,8 @@ RAY_CONFIG(int64_t, log_rotation_backup_count, 5)
 /// notification, in this case we'll wait for a fixed timeout value and then mark it
 /// as failed.
 RAY_CONFIG(int64_t, timeout_ms_task_wait_for_death_info, 1000)
+
+/// The maximum fraction of the object store used for objects pulled from
+/// remote nodes. These objects are arguments of a queued task or objects
+/// requested by `ray.get` or `ray.wait`.
+RAY_CONFIG(float, maximum_object_store_reservation_for_pulled_objects, 0.7)
```

Comment:
```suggestion
RAY_CONFIG(float, task_arg_memory_fraction, 0.7)
```

Reply:
This also includes objects restored from local node etc right? Pulled from remote isn't entirely accurate.

Reply:
Hmm yeah I struggled to think of a name here because it also includes `ray.get` and `ray.wait` objects. Any ideas?

Reply:
I would suggest task arg memory fraction, or if that doesn't seem acceptable then pull_manager_memory_fraction.

It's fine for it to be a little inaccurate on naming as long as it conveys the higher order meaning.